`python312Packages.moviepy` build failure on x86_64-linux as of `21931cc8`

## Issue description
Build of `python312Packages.moviepy` failed on x86_64-linux as of 21931cc8f7cd33304a846cec3d2312f0f714101d. This is currently breaking `python3Packages.wandb`.

```
error: builder for '/nix/store/ikfspf11rq8lh2zvmg3xhjp5r19vr8vl-python3.12-moviepy-2.1.2.drv' failed with exit code 1;
       last 10 log lines:
       >   File "/nix/store/26yi95240650jxp5dj78xzch70i1kzlz-python3-3.12.9/lib/python3.12/threading.py", line 655, in wait
       >     signaled = self._cond.wait(timeout)
       >   File "/nix/store/26yi95240650jxp5dj78xzch70i1kzlz-python3-3.12.9/lib/python3.12/threading.py", line 359, in wait
       >     gotit = waiter.acquire(True, timeout)
       > +++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
       > ----------------------------- Captured stderr call -----------------------------
       > frame_index:  73%|███████▎  | 388/528 [00:57<00:16,  8.46it/s, now=None]
       > =========================== short test summary info ============================
       > FAILED tests/test_issues.py::test_issue_1682 - Failed: Timeout >60.0s
       > =========== 1 failed, 200 passed, 45 deselected in 130.84s (0:02:10) ===========
       For full logs, run 'nix log /nix/store/ikfspf11rq8lh2zvmg3xhjp5r19vr8vl-python3.12-moviepy-2.1.2.drv'.
error: 1 dependencies of derivation '/nix/store/gr8sd9b6788yxpf60m2rszqapqawqvq1-python3.12-wandb-0.19.6.drv' failed to build
```

[full build log](https://github.com/samuela/nixpkgs-upkeep/actions/runs/13555949874)

python3Packages.wandb maintainers cc: @samuela
python312Packages.moviepy maintainers cc: 
Other cc: n/a

### Steps to reproduce
1. Checkout nixpkgs at commit 21931cc8f7cd33304a846cec3d2312f0f714101d
2. Run `nix-build -A python312Packages.moviepy`

## Technical details
```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.0-1021-azure, Ubuntu, 24.04.2 LTS (Noble Numbat), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.16.1`
 - nixpkgs: `/nix/store/ifwi3bmfp1l5rrahs5qvnmpv7xq662sz-source`
```

Contents of `~/.config/nixpkgs/config.nix`:
```
{ allowUnfree = true; cudaSupport = true; cudaCapabilities = ["8.0"]; }
```

## Misc.
This issue was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/13555949874) that created this issue.
- Internal tags: 5315510868795846144 486379699869199481

--- a/pkgs/development/python-modules/moviepy/default.nix
+++ b/pkgs/development/python-modules/moviepy/default.nix
@@ -71,8 +71,8 @@ buildPythonPackage rec {
     pytestCheckHook
   ] ++ lib.flatten (lib.attrValues optional-dependencies);
 
-  # See https://github.com/NixOS/nixpkgs/issues/381908.
-  pytestFlagsArray = [ "--timeout=60" ];
+  # See https://github.com/NixOS/nixpkgs/issues/381908 and https://github.com/NixOS/nixpkgs/issues/385450.
+  pytestFlagsArray = [ "--timeout=600" ];
 
   pythonImportsCheck = [ "moviepy" ];
 

